packages/help/src/Help.tsx

```diff
@@ -1,29 +1,40 @@
 import React, { createContext, useContext, useState, useEffect, useCallback } from 'react';
-import PropTypes from 'prop-types';
 import avMessages from '@availity/message-core';
 import Icon from '@availity/icon';
 
-const HelpContext = createContext();
+type THelp = {
```

Comment:
Is this "T" prefix an old convention for type names or something? Is it something we want to keep?